include/avif/avif.h

```diff
@@ -1567,8 +1574,8 @@ typedef struct avifEncoder
     // Version 1.0.0 ends here.
     // --------------------------------------------------------------------------------------------
 
-    // Defaults to AVIF_HEADER_FULL
-    avifHeaderFormat headerFormat; // Changeable encoder setting.
+    // Defaults to AVIF_HEADER_DEFAULT
+    avifHeaderFormatFlags headerFormat; // Changeable encoder setting.
```

Comment:
For binary (ABI) compatibility, we need to assert that `sizeof(avifHeaderFormatFlags) == sizeof(avifHeaderFormat)`.

Reply:
Done in a test.